### DISCUSSION TITLE
thanhle.blog/blog/lam-viec-tao-ra-value

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#41] [Ideas]
# thanhle.blog/blog/lam-viec-tao-ra-value

Tại sao real-value lại quan trọng

https://thanhle.blog/blog/lam-viec-tao-ra-value

**Comment by nguyentrungson97** [comment]
gamefi đua chó =))

**Comment by nguyenhoaikhang37** [comment]
tỉ năm vào thăm blog mới thấy anh Thành ra thêm bài mới, bữa vào còn bị lỗi tưởng anh stuck làm blogger luôn gòi =)) bài viết có các ví dụ minimal đọc thấm nhanh cực luôn anh

**Reply by thanhlmm** [reply]
:> hic tỉ năm mới vô bùn :)))

**Comment by hai123vn** [comment]
đệt hôm bữa em vô check thấy web anh bị die, tưởng die luôn ai dè anh ra bài mới nè 😂

**Comment by nthang1195** [comment]
hay bạn ạ, hồi này ko thấy bạn ra bài mới

**Comment by HongLaoCai** [comment]
Vào blog thấy bạn giỏi thật
